### DISCUSSION TITLE
New IP for Incognito

### DISCUSSION THREAD

**Discussion by lexaryt** [3kh0/ext-remover#680] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I found an alternative IP to use trying to disable all extensions (aka using incognito)

Not only can you use 158.101.114.159, you can also use:
`52.207.185.90`

This does not work on WiFi network with no password.

**Comment by AshtonDavies** [comment]
Bless your soul for finding another incognito DNS.

**Reply by lexaryt** [reply]
No problem!

**Reply by trentonk23** [reply]
LOCE U BRUH

**Reply by trentonk23** [reply]
bro it dont work....

**Reply by Aka-but-nice** [reply]
There's only one my guy.
